vocabularies/Common.xml

```diff
@@ -553,6 +553,27 @@ The minimum structure is
         </Annotation>
       </Term>
 
+      <Term Name="SAPObjectType" Type="Common.SAPObjectTypeType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The SAP Object Type represented by the annotated entity type" />
+        <Annotation Term="Core.LongDescription">
+          <String>SAP Object Types are a generalization of Business Object, Technical Object, Configuration Object, and Analytical Object.</String>
+        </Annotation>
+      </Term>
+
+      <ComplexType Name="SAPObjectTypeType">
```

Comment:
Why is this a complex type? Do we envisage more properties in the future?

Reply:
Yes, whatever is useful

Reply:
Do you envisage properties of the SAPObjectType? Then you would have to specify them consistently in every occurrence of `Common.SAPObjectType`. Or what kind of property?

Reply:
The `SAPObjectTypeReference` only names the object type referenced by a property.

The `SAPObjectType` marks an entity type as a representation of an SAP Object Type, and this may have additional useful information, for example the category.

Once we have a publicly available repository of SONTs, we would just add a link to it instead of copying a snapshot of the publicly available information.

Reply:
What if one entity type is annotated
```json
"@Common.SAPObjectType": {"Name": "SalesOrder", "Category": "internal"}
```
and another entity type (in a different service, say) as
```json
"@Common.SAPObjectType": {"Name": "SalesOrder", "Category": "external"}
```
? What's then the category of that SAPObjectType?

Reply:
Since this information is to be pulled automatically from the non-public SONT repository, the newer file has the newer content.

Reply:
We could solve this problem if we define SAPObjectTypes in a separate CSDL document.
```json
{"com.sap.objecttypes.SalesAndDistribution": {
 "$Alias": "SD_SOT",
 "@Common.SAPObjectType#SalesOrder": {"Name": "SalesOrder", "Category": "internal"}
}}
```
Here, the annotation applies to the schema. Then we annotate the entity type by referencing that annotation:
```json
"@Common.SAPObjectType": {"$Path": "SD_SOT@Common.SAPObjectType#SalesOrder"}
```

Reply:
Let's have a meeting, I think this discussion is heading in the wrong direction, far away from where we wanted to go.